File: Grasshopper - Grade book.ts
export function getGrade(s1: number, s2: number, s3: number): string {
  let avg = (s1+s2+s3)/3
  if(avg>=90){
    return 'A'
  }else if(avg>=80){
    return 'B'
  }else if(avg>=70){
    return 'C'
  }else if(avg>=60){
    return 'D'
  }else{
    return 'F'
  }
}

File: Grasshopper - Grade book.dart
String getGrade(int a, int b, int c) {
  final score = (a+b+c)/3;
  if(score>=90 && score<=100){
    return 'A';
  }else if(score>=80 && score<90){
    return 'B';
  }else if(score>=70 && score<80){
    return 'C';
  }else if(score>=60 && score<70){
    return 'D';
  }
  return 'F';
}
